pom.xml

```diff
@@ -178,6 +178,7 @@
               <excludes>
                 <exclude>**/target/**</exclude>
                 <exclude>docs/**</exclude>
+                <exclude>src/ci/docker/**</exclude>
```

Comment:
this shouldn't be needed anymore, now that the copyright+license headers have been aded.